src/EnergyPlus/CMakeLists.txt

```diff
@@ -301,7 +301,7 @@ SET( SRC
   Humidifiers.cc
   Humidifiers.hh
 )
-
+ 
```

Comment:
Really! Please turn on "remove trailing spaces on save" mode in your editors folks.